sdk/eventhubs/event-hubs/README.md

```diff
@@ -82,7 +82,7 @@ export DEBUG=azure:event-hubs:error,azure-amqp-common:error,rhea-promise:error,r
 
 ## Examples ##
 
-Please take a look at the [examples](https://github.com/Azure/azure-sdk-for-js/tree/master/packages/%40azure/eventhubs/client/examples) directory for detailed examples.
+Please take a look at the [examples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhubs/event-hubs/samples) directory for detailed examples.
```

Comment:
look at the `samples` directory?

Reply:
The launch.json file has a reference to the `examples` folder which also needs to change to `samples`

Reply:
samples readme.md files also contain references to `example` folder name.